data/src/main/java/com/microsoft/azure/kusto/data/KustoResultSetTable.java

```diff
@@ -506,42 +415,26 @@ public BigDecimal getBigDecimal(int columnIndex) {
         return new BigDecimal(getString(columnIndex));
     }
 
-    @Override
     public BigDecimal getBigDecimal(String columnName) {
         return getBigDecimal(findColumn(columnName));
     }
 
-    @Override
     public boolean isBeforeFirst() {
         return currentRow == null;
     }
 
-    @Override
     public boolean isAfterLast() {
         return currentRow == null && !rowIterator.hasNext();
     }
 
-    @Override
-    public boolean isFirst() throws SQLException {
-        throw new SQLFeatureNotSupportedException("Method not supported");
-    }
-
-    @Override
     public boolean isLast() {
         return currentRow != null && !rowIterator.hasNext();
     }
 
-    @Override
     public void beforeFirst() {
         rowIterator = rows.iterator();
     }
 
-    @Override
-    public void afterLast() {
-        while (next()) ;
```

Comment:
Why was this removed (not deprecated it not implemented)?

Reply:
I dont think its relevant to this SDK usage - i think the only reason to use this is if you provide two ways iterator - so you can go afterLast and do while(previous)